src/config/api/polkadot/connectApi.ts

```diff
@@ -61,7 +61,7 @@ export async function connectApi(endpoint: string, networkIdx: number) {
   let extensions: InjectedExtension[] = [];
 
   let typeDefinitions = null;
-  if (networkIdx === endpointKey.SHIDEN) {
+  if (networkIdx === endpointKey.SHIDEN || networkIdx === endpointKey.SHIBUYA) {
```

Comment:
instead of having a list of `if` conditions, what about embedding the type definitions on the `ChainProvider` object? https://github.com/PlasmNetwork/astar-apps/pull/9/files#diff-bbb15aefae956e4e9ba6a9527a96e3f2693a2d247e6c2235d13d8f25f3274259R21

for example, we can add a new property to the interface
```ts
interface ChainProvider {
  networkAlias: string;
  displayName: string;
  info?: string;
  // ...
  typeDef: RegistryTypes;
}
```